src/config/dynamic-theme-fixes.config

```diff
@@ -3902,6 +3902,38 @@ INVERT
 
 ================================
 
+jumia.com
+jumia.co
+jumia.ma
+jumia.ci
+jumia.dz
+jumia.ug
+jumia.sn
+zando.co.za
+
+INVERT
+svg.ic[role="img"]:not([aria-label="Zando logo"])
+.logo > a > img
+.inbox > a:not(:nth-of-type(1)) > svg
+img[src*="jumia-group-logo.png"]
```

Comment:
Logo in group.jumia.com
![Jumia group logo](https://user-images.githubusercontent.com/14346945/100476614-b4498980-30ee-11eb-88b5-e17e60b9deea.jpg)
![Jumia group logo (2)](https://user-images.githubusercontent.com/14346945/100476616-b57ab680-30ee-11eb-8c5e-24c51469ba52.jpg)